test/e2e/pkg/client/pod.go

```diff
@@ -99,3 +98,18 @@ func (w *K8sClient) isPodRunning(podName, namespace string) wait.ConditionFunc {
 		return false, nil
 	}
 }
+// return the pod name from the dev user namespace (config.DevNameSpace)
+func (w *K8sClient) GetPodNameFromUserNameSpaceByLabel(selector string)  string {
+	podList, err := w.ListPods(config.DevNameSpace, selector)
+
+	if err != nil {
+		log.Fatal("Cannot obtain pods from developer namespace " + err.Error())
+	}
+	if len(podList.Items) == 0 {
+		log.Fatal("Cannot find pods in developer namespace with selector " + selector)
```

Comment:
Same comment here, should we log with the config.DevNamespace here

Reply:
Done